west.yml

```diff
@@ -68,7 +68,7 @@ manifest:
       revision: a1ec761014740a08699720298dd37ad4da360840
       path: modules/hal/microchip
     - name: hal_silabs
-      revision: 78da967feeac0d51219ef733cc3ccf643336589f
+      revision: pull/8/head
```

Comment:
You can make this a github "Draft" to make sure `pull/8/head` can never get merged.

Reply:
Thank you for the hint